src/test/compile-fail/invalid-module-declaration.rs

```diff
@@ -0,0 +1,20 @@
+// Copyright 2016 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+// ignore-tidy-linelength
+
+// error-pattern: cannot declare a new module at this location
+// error-pattern: maybe move this module
```

Comment:
> I removed the `strip_prefix`, so now it's the full path starting from `/`

I believe it was the full path starting from `/` even with `strip_prefix` -- that is, the `strip_prefix` stuff didn't make a difference in this case.

> Therefore, if I put it, it won't work because it'll be mine

Good point -- the global path will depend on what platform is running the tests.

It looks like we'll need a `ui` test, or if that doesn't work either a `run-make` test. The current test isn't useful though since it would have passed before the PR.